docs/source/features/sheet_management/dataframe.rst

```diff
@@ -4,3 +4,145 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the 2-dimensional data with columns of Protex schema.
+
+************************
+ Initialize a DataFrame
+************************
+
+Initialize a Dataframe from a list of dicts:
+
+.. code:: python
+
+   >>> from graviti import DataFrame
+   >>> data = [
+   ...    {"filename": "a.jpg"},
+   ...    {"filename": "b.jpg"},
+   ...    {"filename": "c.jpg"},
+   ... ]
+   >>> df = DataFrame(data)
+   >>> df
+      filename
+   0  a.jpg
+   1  b.jpg
+   2  c.jpg
+
+Initialize a DataFrame with Multi-level column names:
+
+.. code:: python
+
+   >>> from graviti import DataFrame
+   >>> data = [
+   ...      {"attribute": {"weather": "sunny", "color": "red"}},
+   ...      {"attribute": {"weather": "rainy", "color": "black"}},
+   ...      {"attribute": {"weather": "sunny", "color": "white"}},
+   ... ]
+   >>> df = DataFrame(data)
+   >>> df
+      attribute
+      color      weather
+   0  red        sunny
+   1  black      rainy
+   2  white      sunny
+
+Initialize a DataFrame with nested DataFrame construction:
+
+.. code:: python
+
+   >>> from graviti import DataFrame
+   >>> data = [
+   ...    {"points": [{"xmin": 1, "ymin": 3}, {"xmin": 5, "ymin": 8}]},
+   ...    {"points": [{"xmin": 6, "ymin": 10}]},
+   ...    {"points": [{"xmin": 1, "ymin": 3}, {"xmin": 5, "ymin": 8}, {"xmin": 1, "ymin": 9}]},
+   ... ]
+   >>> df = DataFrame(data)
+   >>> df
+      points
+   0  DataFrame(2, 2)
+   1  DataFrame(1, 2)
+   2  DataFrame(3, 2)
+   >>> df["points"][0]
+      xmin  ymin
+   0  1     3
+   1  5     8
+
+****************
+ Read DataFrame
+****************
+
+Read data by row:
+
+.. code:: python
+
+   df.loc[0]
+
+Read data by column:
+
+.. code:: python
+
+   df[f"{COLUMN_NAME}"]
+
+Read a DataFrame cell:
+
+.. code:: python
+
+   df.loc[0][f"{COLUMN_NAME}"]
+   df[f"{COLUMN_NAME}"][0]
+
+****************
+ Edit DataFrame
+****************
+
+Modify Column Content:
+
+.. code:: python
+
+   df[f"{COLUMN_NAME}"] = [1, 4, 9]
```

Comment:
Now we only support changing the column content of local DataFrame. do not need to add this method now.